### DISCUSSION TITLE
auto populate options from dropdown to multiselect

### DISCUSSION THREAD

**Discussion by gixxy22** [craftcms/cms#9877] [Ideas] [answered]
so upon creating many many options for a dropdown list over several fields, i then realised they needed to be multi-select fields instead, however upon changing the field to a multi-select, i have to re-input the data again, the data is already available and they are similar fields, would it be beneficial to auto populate if the field was previously dropdown/multiselect?

not a massive deal but could come in handy for others new to craft possibly.

**Comment by brandonkelly** [comment, marked as answer]
They store data in different formats (Dropdown fields store the raw selected option value; Multi-select fields store all selected values as JSON.

You could update the existing values by running this SQL query:

```sql
UPDATE content SET myFieldColumnName = CONCAT('["', myFieldColumnName, '"]')
WHERE myFieldColumnName IS NOT NULL
AND myFieldColumnName <> ''
AND myFieldColumnName NOT LIKE '[%'
```
